Project.toml

```diff
@@ -27,7 +27,7 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 
 [compat]
 ClimaAtmos = "0.15"
-ClimaComms = "0.3, 0.4"
+ClimaComms = "0.5"
```

Comment:
Do we want this to be `"0.4, 0.5"` now?

Reply:
I tried both ways, and it was not necessary to keep the previous version. Actually, it would prevent other package updates for compatibility constraints